src/components/sidebar.scss

```diff
@@ -69,6 +69,7 @@
   --__op-sidebar-width: var(--_op-sidebar-drawer-width);
   --__op-sidebar-brand-width: var(--_op-sidebar-drawer-brand-width);
 
+  z-index: var(--op-z-index-drawer);
```

Comment:
@Jeremy-Walton This works, but should we introduce a specific token for `--op-z-index-sidebar`?

Reply:
This token is meant to be specific for this. Updating to be `--op-z-index-sidebar`